packages/cli/src/controllers/me.controller.ts

```diff
@@ -101,7 +99,6 @@ export class MeController {
 		this.authService.issueCookie(res, user, req.browserId);
 
 		const fieldsChanged = Object.keys(payload);
-		this.internalHooks.onUserUpdate({ user, fields_changed: fieldsChanged });
```

Comment:
Pre-existing bug: Even when updating a single field, all fields exist in the payload, so all fields are being reported as having changed. To be fixed separately.

Reply:
I'll follow up on this, won't forget